packages/core/content-type-builder/admin/src/components/DataManagerProvider/reducer.js

```diff
@@ -40,6 +40,34 @@ const reducer = (state = initialState, action) =>
   // eslint-disable-next-line consistent-return
   produce(state, (draftState) => {
     switch (action.type) {
+      case actions.ADD_CUSTOM_FIELD_ATTRIBUTE: {
```

Comment:
Can you keep the alphabetic order, since there is a lot of case I believe it makes it clearer to follow...

Reply:
https://github.com/strapi/strapi/pull/14465/commits/5e8cb0f63bd08a35228f5d1b37bd211bc4920cc8